enhancements/update/standalone-worker-surge.md

```diff
@@ -0,0 +1,295 @@
+
+Documentation for these configuration options can be found in HyperShift's API reference: 
+- https://hypershift-docs.netlify.app/reference/api/#hypershift.openshift.io/v1beta1.NodePoolManagement exposes `MaxSurge`.
+- https://hypershift-docs.netlify.app/reference/api/#hypershift.openshift.io/v1beta1.NodePoolSpec exposes `NodePoolTimeout`
+
+Example `MachineConfigPool` including both `NodeDrainTimeout` and a `MaxSurge` setting:
+```yaml
+kind: MachineConfigPool
+spec:
+  # Existing spec fields are not shown.    
+
+  # Adopted from NodePool to create consistency and further our goal
+  # to improve the reliability of worker updates.
+  nodeDrainTimeout: 10m
+      
+  # New policy analog to NodePool.NodePoolManagement.
+  machineManagement:
+    upgradeType: "Replace"
+    replace:
+      strategy: "RollingUpdate"
+      rollingUpdate:
+        maxUnavailable: 0
+        maxSurge: 4
+```
+
+Like HCP, `UpgradeType` will support:
+- `InPlace` where no additional nodes are brought online to support draining workloads.
+- `Replace` where new nodes will be brought online with `MaxSurge` support.
+
+#### InPlace Update Type
+
+The `InPlace` update type is similar to `MachineConfigPools` traditional behavior where a 
+user can configure the `MaxUnavailable` nodes. This approach assumes the number (or percentage)
+of nodes specified by `MaxUnavailable` can be drained simultaneously with workloads finding
+sufficient resources on other nodes to avoid stalled drains.
+
+```yaml
+kind: MachineConfigPool
+spec:
+  # Existing spec fields are not shown.    
+
+  machineManagement:
+    upgradeType: "InPlace"
+    inPlace:
+      maxUnavailable: 10%
+```
+
+Nodes are rebooted after they are drained.
+
+#### Replace Update Type
+
+The `Replace` update type removes old machines and replaces them with new instances. It supports
+two strategies:
+- `OnDelete` where a new machine is brought online only after the old machine is deleted.
+- `RollingUpdate` which supports the `MaxSurge` option.
+
+```yaml
+kind: MachineConfigPool
+spec:
+  # Existing spec fields are not shown.    
+
+  machineManagement:
+    upgradeType: "Replace"
+    replace:
+      strategy: "RollingUpdate"
+      rollingUpdate:
+        maxUnavailable: 0
+        maxSurge: 4
+```
+
+`MaxSurge` applies independently to each associated MachineSet. For example, if three MachineSets are associated
+with a MachineConfigPool, and `MaxSurge` is set to 4, then it is possible for the cluster to surge up to 12 nodes
+(4 for each of the 3 MachineSets).
+
+### Topology Considerations
+
+Multi-AZ (availability zone) clusters function by using one or more `MachineSets` per zone. In order for this enhancement
+to work across all zones, all such `MachineSets` should be associated with a `MachineConfigPool` with well considered
+values for `MaxSurge` and `NodeDrainTimeout`.
+
+Each `MachineSet` associated with a `MachineConfigPool` will be permitted to scale by the `MaxSurge` number of nodes. 

```

Comment:
This will be confusing for end users, we may want to try and name the field so that it indicates this better than what it does today.

Also, this really makes me think we want to reverse this and look at this as a CAPI problem rather than an MCP problem